perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -14,50 +15,174 @@ has 'whitespace' => (
     default => sub { Integrations::Whitespace->new(%{shift->feature('whitespace')}) },
 );
 
+sub postcode_database_file {
+    FixMyStreet->path_to('../data/bexley-ww-postcodes.sqlite');
+}
+
 sub bin_addresses_for_postcode {
     my ($self, $postcode) = @_;
 
-    my $addresses = $self->whitespace->GetAddresses($postcode);
+    return [] unless -e postcode_database_file();
```

Comment:
I think it might be worth putting in a `die` here if there's no database file, don't think much else is going to work without it, so might as well fail loudly if it's missing!